### PR TITLE
Add default for ERC20 properties of curve gauges

### PR DESCRIPTION
**Pull request by yabirgb**
Old curve gauges don't have the symbol, name, decimals methods. I asked in telegram and the default is 18. This PR sets default values for this attributes to avoid errors at the moment of populating the curve cache

Closes #(issue_number)

## Checklist

- [ ] The PR modified the frontend, and updated the [user guide](https://github.com/rotki/rotki/blob/develop/docs/usage_guide.rst) to reflect the changes.

### CODE REVIEW

File: rotkehlchen/chain/evm/decoding/curve/decoder.py [hunk 1]
```diff
@@ -87,7 +87,7 @@ def __init__(
         ReloadablePoolsAndGaugesDecoderMixin.__init__(
             self,
             evm_inquirer=evm_inquirer,
-            cache_type_to_check_for_freshness=CacheType.CURVE_LP_TOKENS,
+            cache_type_to_check_for_freshness=CacheType.CURVE_POOL_ADDRESS,
```
Comment (LefterisJP) [inline rotkehlchen/chain/evm/decoding/curve/decoder.py:90]: so this was a bug?

File: rotkehlchen/chain/evm/decoding/curve/curve_cache.py [hunk 2]
```diff
@@ -265,6 +265,9 @@ def _ensure_curve_tokens_existence(
                         token_kind=EvmTokenKind.ERC20,
                         weight=ONE,
                     )],
+                    decimals=18,  # 18 is the default for old gauges that don't implement the decimals method https://t.me/curvefi/654915  # noqa: E501
```
Comment (LefterisJP) [inline rotkehlchen/chain/evm/decoding/curve/curve_cache.py:268]:
```suggestion
                    decimals=18,  # all gauges have 18 decimals https://t.me/curvefi/654915. Hard code since some contracts have no name/symbol/decimals.  # noqa: E501
```

File: rotkehlchen/chain/evm/decoding/curve/curve_cache.py [hunk 3]
```diff
@@ -493,13 +496,14 @@ def query_curve_data(
...
     with GlobalDBHandler().conn.read_ctx() as cursor:
-        existing_pools = [
-            string_to_evm_address(address)
-            for address in globaldb_get_general_cache_like(
-                cursor=cursor,
-                key_parts=(cache_type, str(inquirer.chain_id.serialize_for_db())),
+        existing_pools = {  # query the pools that we already have in the db
+            string_to_evm_address(address[0])
+            for address in cursor.execute(
+                'SELECT value FROM unique_cache WHERE key LIKE ?',
```
Comment (LefterisJP) [inline rotkehlchen/chain/evm/decoding/curve/curve_cache.py:502]: why this change?

**Review by LefterisJP — COMMENTED** [review]
Ah also @yabirgb we will need a changelog here

File: rotkehlchen/chain/evm/decoding/curve/curve_cache.py (hunk 3 above)
Comment (yabirgb) [reply rotkehlchen/chain/evm/decoding/curve/curve_cache.py:502]: We need to access the unique_cache instead of the general cache

File: rotkehlchen/chain/evm/decoding/curve/decoder.py (hunk 1 above)
Comment (yabirgb) [reply rotkehlchen/chain/evm/decoding/curve/decoder.py:90]: yes, reading the wrong cache tyype

File: rotkehlchen/assets/utils.py [hunk 4]
```diff
@@ -215,6 +215,9 @@ def get_or_create_evm_token(
         encounter: TokenEncounterInfo | None = None,
         coingecko: str | None = None,
         cryptocompare: str | None = None,
+        fallback_decimals: int | None = None,
```
Comment (LefterisJP) [inline rotkehlchen/assets/utils.py:218]: Why do this? In our usecase it's not a fallback.

In our usecase they are all the same, no ?

If provided instead just don't query the chain at all. Why query, only to see it's missing and then use the given one? That makes no sense.

File: rotkehlchen/tests/unit/test_tokens.py [hunk 5]
```diff
@@ -261,3 +265,75 @@ def test_flaky_binding_parameter_zero(
     with database.conn.read_ctx() as cursor:
         for address in ethereum_accounts:
             database.get_tokens_for_address(cursor, address, SupportedBlockchain.ETHEREUM)
+
+
+@pytest.mark.parametrize('number_of_eth_accounts', [1])
+def test_old_curve_gauge(ethereum_inquirer: 'EthereumInquirer'):
+    """Test that querying new and old gauges get the data correctly
```
Comment (LefterisJP) [inline rotkehlchen/tests/unit/test_tokens.py:272]:
```suggestion
    """Test that querying new and old gauges get the data correctly.
```

File: rotkehlchen/tests/unit/test_tokens.py [hunk 6]
```diff
@@ -261,3 +265,75 @@ def test_flaky_binding_parameter_zero(
...
+    Old one should peak the default values provided and the new one should
```
Comment (LefterisJP) [inline rotkehlchen/tests/unit/test_tokens.py:273]:
```suggestion
    Old one should pick the default values provided and the new one should
```

**Review by LefterisJP — APPROVED** [review]
LGTM
